source/guide_snibox.rst

```diff
@@ -0,0 +1,216 @@
+.. highlight:: console
+
+.. author:: Maximilian F. Kapfer <maximilian.kapfer@gmail.com>
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/snibox.png
+      :align: center
+
+######
+Snibox
+######
+
+Snibox is a self-hosted snippet manager. Developed to collect and organize code snippets. Supports various programming languages, markdown, plain text.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * PostgreSQL_
+  * Ruby_
+  * supervisord_
+  * domains_
+
+
+License
+=======
+
+Snibox is released under the MIT License. All relevant legal information can be found here:
+
+  * https://opensource.org/licenses/MIT
+
+
+Prerequisites
+=============
+For this service we need to use Ruby in the version 2.6 and PostgreSQL in the version 12.
+
+::
+
+ [isabell@stardust ~]$ uberspace tools version list ruby
+    - 2.5
+    - 2.6
+    - 2.7
+    - 3.0
+
+ [isabell@stardust ~]$ uberspace tools version use ruby 2.6
+    Selected ruby version 2.6
+    The new configuration is adapted immediately. Patch updates will be applied automatically.
+
+ [isabell@stardust ~]$ uberspace tools version list postgresql
+    - 10
+    - 11
+    - 12
+    - 13
+
+ [isabell@stardust ~]$ uberspace tools version use postgresql 12
+ Using 'postgresql' version: 13
+
+Next, we need a running PostgreSQL database. A detailed description can be found in the
+`PostgreSQL Guide <https://lab.uberspace.de/guide_postgresql.html>`_
+
+If you want to use Snibox with your own domain you need to setup your domains_ first:
+
+.. include:: includes/web-domain-list.rst
+
+Installation
+============
+
+Step 1 - Download & Extract
```

Comment:
Please don't use numbered steps since this makes editing the guide later very cumbersome. Just strip the `Step X -` from all of them and leave the names.